test/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+import pytest
+
+
+@pytest.fixture
+def test_dir():
+    return os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def fixture_dir(test_dir):
+    return f"{test_dir}/fixtures"
```

Comment:
Ideally we would join paths with `os.path.join()` instead of hardcoding slashes (except for URLs of course), but I agree it does not make much of a difference since we're never going to run it on Windows ;-)